doc/source/frame.rst

```diff
@@ -0,0 +1,34 @@
+Frame
+=====
+
+PIMS returns images as ``Frame`` objects. Frames can be treated precisely
+the same as numpy arrays.
+
+They are a subclass of ``numpy.ndarray``, adding two new attributes:
+
+* ``frame_no``, an integer
+* ``metadata``, a dictionary
+
+These can be used by the PIMS readers to provide any metadata stored in the
+image files. Setting these attriburtes is optional.
+
```

Comment:
For convenience, Frame objects in IPython notebook are displayed as actual images. 3D images ("Z stacks") are displayed as a stack of images. The user can scroll through the images with the scroll wheel.